CONP_DATS_fields.md

```diff
@@ -14,10 +14,10 @@ We use the `DATS.json` file format to store metadata describing CONP datasets.
 |Size|distributions - size|**REQUIRED** Total size of the dataset.| The quantity of space required on disk (or other medium) for this dataset.|
 |Units|distributions - unit|**REQUIRED** Unit in which the size is measured.(*example*: GB)|Unit in which the size is measured.(KB -> KiloByte, MB -> MegaByte, GB -> GigaByte, TB -> TeraByte and PB -> PetaByte). |
 |Landing Page|distributions - access - landingPage|**REQUIRED** Web address (URL) where the original dataset can be found. Default value points to the README.md for the dataset. |A URL (Web page) that contains information about the associated dataset (with a link to the dataset) or a direct link to the dataset itself. |
-|Authorizations|distributions - access - authorizations| **REQUIRED**Types of verification that accessing the resource is allowed. Authorization occurs before successful authentication and refers to the process of obtaining approval to use a data set. Ideally specified from a controlled vocabulary or ontology.|This must be one of "Public", "Registered" or "Private". When this field is absent the value will be treated as "Public". "Public" = available to anyone; "Registered" = available to user authorized by the creator; "Private" = available only to the creator.|
```

Comment:
Authorization should not be removed from the documentation as this is a separate field than privacy that goes under the distribution section of the DATS file while privacy is directly at the root of the DATS file.